libs/testing/src/mailtrap.service.ts

```diff
@@ -162,11 +178,64 @@ export class MailtrapService {
    * https://api-docs.mailtrap.io/docs/mailtrap-api-docs/bcf61cdc1547e-send-email-early-access
    */
   async sendEmail(inboxId: MailtrapInboxId, payload: IEmailWithText): Promise<ISendEmailResponseDto> {
-    const url = this.buildSandboxApiUrl(`/send/${inboxId}`);
+    const url = this.buildSandboxApiUrl(`${BASE_PATH}/send/${inboxId}`);
 
     const response = await this.postCall<IEmailWithText, ISendEmailResponseDto>(url, payload);
     const { data } = response;
 
     return data;
   }
+
+  /**
+   * Due the async nature of sending an email and being processed we use this poller to try to
+   * know when the email has arrived.
```

Comment:
Also this is the suggested solution from Mailtrap for now. They are working on implementing a WebSocket solution. No E.T.A.